### DISCUSSION TITLE
What is the need for flow in your application?

### DISCUSSION THREAD

**Discussion by oliverpatrick** [amilajack/popcorn-time-desktop#616] [Q&A]
Why is flow needed for this app? Is it necessary for css?

**Comment by amilajack** [comment]
It was removed in https://github.com/amilajack/popcorn-time-desktop/pull/607

**Reply by oliverpatrick** [reply]
In the internals there is a flow folder, is there any point to it?

**Reply by amilajack** [reply]
The project is now completely migrated to typescript
